README.md

```diff
@@ -124,7 +126,9 @@ Wait until all current metrics are sent to the server. <br>
 We recommend using it at the end of lambda execution to make sure all metrics are sent.
 
 ```ts
-await monitor.flush(timeout: number = 2000)
+import { getGlobalInstance } from 'monitored';
+
+await getGlobalInstance().flush(timeout: number = 2000)
```

Comment:
```suggestion
const flushTimeout: number = 2000;
await getGlobalInstance().flush(flushTimeout)
```

I assume `flush` does not take `({timeout: 2000})` as input, so let's make it so that the code is at least valid syntax